coalib/output/ConsoleInteraction.py

```diff
@@ -94,6 +94,7 @@ def highlight_text(no_color, text, lexer=TextLexer(), style=None):
 
 
 def color_letter(console_printer, line):
+    endc = '\033[0m'
```

Comment:
Can you make this variable a level higher? I don't like seeing it repeatedly all over the code :/